services/wallet/reactor.go

```diff
@@ -93,6 +87,19 @@ func (r *Reactor) Start(accounts []common.Address) error {
 		feed:        r.feed,
 		safetyDepth: reorgSafetyDepth,
 	}
+
+	return ctl
+}
+
+// Start runs reactor loop in background.
+func (r *Reactor) Start(accounts []common.Address) error {
+	r.mu.Lock()
+	defer r.mu.Unlock()
+	if r.group != nil {
+		return errors.New("already running")
```

Comment:
consider making error a "constant".

Also this could gain from using `sync.Once` but it's probably outside the scope of this PR

Reply:
> consider making error a "constant".

https://github.com/status-im/status-go/blob/c8a911ebd1e5f6c567d8ff6f3700309b988f0d42/services/wallet/reactor.go#L79

ok 

> Also this could gain from using sync.Once

yep it could, if I knew what it means :D